internal/services/appconfiguration/app_configuration_feature_resource.go

```diff
@@ -42,6 +44,15 @@ type FeatureResourceModel struct {
 	TargetingFilters     []TargetingFilterAudience    `tfschema:"targeting_filter"`
 }
 
+func (FeatureResource) StateUpgraders() sdk.StateUpgradeData {
```

Comment:
could you explain why we need a state migration here? no existing resources should have a / ?

Reply:
there is a possibility existing resources have "/" in its id, and the state file will be "broken" like the issue described #19437. I think we add a state migration here will help

Reply:
removed in new commit